src/components/SearchableDropdown/SearchableDropdown.stories.tsx

```diff
@@ -1,34 +1,28 @@
 import { useState } from "react";
-import SearchableDropdown from ".";
+import { StoryObj } from "@storybook/react";
+import SearchableDropdown, { SearchableDropdownProps } from ".";
 
 export default {
-  title: "Components/Searchable Dropdown",
   component: SearchableDropdown,
 };
 
```

Comment:
I think the `CustomOption` component might have been missed in this file (to turn into a `StoryObj` thing)